app/fullService/api/getNetworkStatus.ts

```diff
@@ -3,6 +3,9 @@ import axiosFullService, { AxiosFullServiceResponse } from '../axiosFullService'
 
 const GET_NETWORK_STATUS_METHOD = 'get_network_status';
 
+// ideally this would be fetched from the network, but FS can't know current bloc version in offline mode
+export const BLOCK_VERSION = '2';
```

Comment:
I think this can be retrieved from an offline ledger, so long as the ledger service can be run. Something like:
```
use crate::WalletService
let service = WalletService::new(/*params*/) 
(block, _) = service.get_block_object(/*params*/)
let version = block.version
```
Is this doable here? It would be great to not have a magic number, even in a declared constant like this